examples/plotting/line_on_off.py

```diff
@@ -1,4 +1,11 @@
-""" Example demonstrating turning lines on and off - with JS only
+""" Example demonstrating turning lines on and off - with JS only.
+It involves checking and unchecking the checkboxes representing
+the plotted lines to turn the lines on/off.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.layouts.row
+    :refs: :ref:`ug_basic_lines_line_multiple`
+    :keywords: line_on_off, checkbox, CustomJS
```

Comment:
I don't know if line_on_off is a keyword that people would search for - how about we keep it simple and just go with line chart:

```suggestion
    :keywords: line, checkbox, CustomJS
```

Reply:
Executed all the requested changes.